### ISSUE TITLE
new clusters from noise

### ISSUE THREAD

**Issue by vamsibanda** [scikit-learn-contrib/hdbscan#524]
I have trained a model initially on a custom dataset and used the approximate_predict operation to compute a predicted cluster for the new data points every day. Over time, I have collected all the noise examples and found out there is a possibility for new clusters. If I fit the model again it will change the existing clusters. Is there a way I can find new clusters from noise and add them as an extension to the existing model?

**Comment by adilosa** [comment]
I don't think it is possible to add clusters to an existing model, or to train a new model to keep some existing clusters the same (#70). Perhaps the easiest thing would be to keep the first model, train a second model on the noise, and combine the results.
